contrib/enum_metrics/enum_metrics/dbclient.py

```diff
@@ -0,0 +1,223 @@
+from cassandra.cluster import Cluster
+from cassandra.policies import (RetryPolicy)
+from cassandra.query import (PreparedStatement, BoundStatement)
+
+
+class DBClient:
+    session = None
+
+    def connect(self, nodes):
+        cluster = Cluster(
+            contact_points=nodes,
+            default_retry_policy=RetryPolicy()
+        )
+        metadata = cluster.metadata
+        self.session = cluster.connect('DATA')
+        print '\nConnected to cluster: %s' % metadata.cluster_name
+        for host in metadata.all_hosts():
+            print 'Datacenter: %s; Host: %s; Rack: %s' % (host.datacenter, host.address, host.rack)
+
+    def close(self):
+        self.session.cluster.shutdown()
+        self.session.shutdown()
+        print 'Connection closed.'
+
+    def format_results(self, results):
+        """
+        Formats results to a tuple and returns it with just the (key, column1, value) from
+        database results.
+
+        :param results:
+        :return:
+        """
+
+        return [(x.key, x.column1, x.value) for x in results]
+
+
+    def get_metrics_excess_enums(self, metric_name):
```

Comment:
I thought about it when writing the code. It felt like I am writing dynamic SQL. I thought I would rather have simple select queries for each table. Since we are not doing any complex query, may be I could extract that logic to one method.